project_generator/project.py

```diff
@@ -361,48 +364,33 @@ def flash(self, tool):
             project_files = [os.path.join(proj_dic['output_dir']['path'], proj_dic['name'])]
         flash(flasher, proj_dic, self.name, project_files, tool, self.workspace.settings)
 
-    def export(self, tool, copy):
+    def generate_dic(self, tool, copy):
```

Comment:
this function needs to be removed and the magic inside needs to be refactored. There's similar functionality for pgen keywords ($tool$,..), unify them into one class?